### DISCUSSION TITLE
Use Rustdesk as a quicksupport version ?

### DISCUSSION THREAD

**Discussion by Tuxman2** [rustdesk/rustdesk#5508] [Q&A] [answered]
Hello,

is it possible to use the actual rustdesk version as a quicksupport version ?

I ask you this because I would like to have a version comparable to that of teamviewer quicksupport with rustdesk. So that only an IT technician can take control of the remote machine without user intervention. Is this kind of thing currently possible with rustdesk? If yes, how to implement this?

Thanks.

Regards.

**Comment by SergeyMy** [comment]
at least you need to have your owns! available outside servers:
1. Relay Server
2. ID/Rendezvous Server
optional 3.  API Server

**Comment by pixelstuff** [comment]
One thing I like about TeamViewer Quick Support is that it launches and automatically requests administrator level permissions so there's no need later for a technician to ask the customer to later approve a dialog box (or grant extra permission or whatever). It would be nice if the RustDesk installer could run with admin permissions without having to permanently install the software.

**Comment by rustdesk** [comment]
Since this is totally a commercial use scenario, it will be ready in Pro version. Contact our sales@rustdesk.com

**Reply by pixelstuff** [reply]
It's not totally a commercial use scenario, perhaps just mostly. Plenty of family and friends need help with something on rare occasions where installing is overkill (except for the fact that if they don't they have to run back to the computer when a UAC dialog pops up) and uninstalling after the session is a hassle.

Of course they could just leave it installed, but most would never use it themselves.

**Comment by SergeyMy** [comment]
@pixelstuff let me tell you a secret :) rustdesk can work without admin rights and installation!

**Reply by pixelstuff** [reply]
I'm talking about installing apps, drivers, etc. You couldn't do that through Rustdesk unless it was installed.  However TeamViewer Quick Support allows it without installing.

**Reply by rustdesk** [reply]
we can, https://github.com/rustdesk/rustdesk/issues/2155

**Reply by rustdesk** [reply]
@21pages add doc for this please. A seperate page for this under client chapter. including "accept and elevate" and about remote request elevation etc.

**Comment by subnetmasked** [comment]
A "quicksupport"-ish version of RustDesk is the biggest feature that's missing imo. We'll drop teamviewer when/if it becomes available.
Tbh, I think it's going to explode in popularity if this is implemented.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/8981#discussioncomment-10263847
